src/ServiceInsight/Startup/StartupConfigListener.cs

```diff
@@ -40,7 +40,7 @@ public static async Task Start(IEventAggregator eventAggregator, CommandLineArgP
 
         static async Task Handle(IEventAggregator eventAggregator, CommandLineArgParser parser, CancellationToken cancellationToken)
         {
-            using (var pipe = new NamedPipeServerStream("ServiceInsight", PipeDirection.In, 1, PipeTransmissionMode.Byte))
+            using (var pipe = new NamedPipeServerStream($"ServiceInsight-{Environment.UserName}", PipeDirection.In, 1, PipeTransmissionMode.Byte))
```

Comment:
nit: I'm not a big fan of this duplication in two different files. We know the name has to be identical in two places; therefore, we could extract it into a component/static class and consolidate it.

Reply:
Agreed. I'll update it.

Reply:
I took care of it in https://github.com/Particular/ServiceInsight/pull/1064/commits/ad2b694f0243a227146188af91366899203eddc2